rero_ils/modules/loans/api.py

```diff
@@ -42,6 +43,19 @@
 from ..utils import get_ref_for_pid
 
 
+class LoanState(Enum):
+    """Enum class to handle different loan states."""
+
+    CREATED = 'CREATED'
+    PENDING = 'PENDING'
+    ITEM_IN_TRANSIT_FOR_PICKUP = 'ITEM_IN_TRANSIT_FOR_PICKUP'
+    ITEM_IN_TRANSIT_TO_HOUSE = 'ITEM_IN_TRANSIT_TO_HOUSE'
+    ITEM_AT_DESK = 'ITEM_AT_DESK'
+    ITEM_ON_LOAN = 'ITEM_ON_LOAN'
+    ITEM_RETURNED = 'ITEM_RETURNED'
+    CANCELLED = 'CANCELLED'
+
```

Comment:
Why not do the same as `class ItemStatus` and use a `object`? It could be more useful because you can use `LoanState.CREATED` instead of `LoanState.CREATED.value`!

Reply:
I do not understand you point here. `ItemStatus` is very similar and we do not need `.value` here. But perhaps I misunderstood.

Reply:
@jma You asked to change it to `.value` here -> https://github.com/rero/rero-ils/pull/1010#discussion_r431766491

Reply:
You suggested it here: https://github.com/rero/rero-ils/pull/1010#discussion_r431766491 .
But to be consistent, either we use `Enum` for ItemStatus and LoanState or we use `object` for both :wink: 

In my opinion, `Enum` is only useful if you need to list all states in a web interface. It's the only advantage.

Reply:
I will put it back as `object`.